### ISSUE TITLE
Example of confusion created by the host/symbiont triad

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#17045]
Hello, 

Trying to illustrate why I think we should not systematically make the host/symbiont triad: 

PARENT CLASS: GO:0044112  growth in other organism involved in symbiotic interaction
    -> no annotations

But, do we expect the host to grow **in** the symbiont ? The only possibility here is 'growth of symbiont in host' 

I would prefer to obsolete that class, as well as 3 of its children: 

GO:0044110    growth involved in symbiotic interaction
	-> 5 annotations by MTBBASE; should have been 'growth of symbiont in host cell'

GO:0044153 growth on or near surface of other organism involved in symbiotic interaction
 and its child GO:0044151 growth of organism on or near symbiont surface 
 -> no annotations

----
These terms seem sufficient to represent what needs to be captured: 
- growth of symbiont in host
- growth of symbiont in host cell
- growth of symbiont on or near host
- growth of symbiont in host organelle

@jimhu-tamu @mgiglio99 @cmungall @pmasson55 @nsuvarnaiari @jrr-cpt @valwood 
Do you agree to obsolete the 3 terms ? 

Thanks, Pascale

**Comment by cmungall** [comment]
Very good point, I agree

>

**Comment by mgiglio99** [comment]
The problem is that not all symbiotic interactions involve a host and hostee. There are many where the relationship is equal - like a lichen. That’s why we made all of the “other organism” terms and had that structure.

**Comment by ukemi** [comment]
parasitism, mutualism, commensalism if I remember correctly from undergraduate biology.

**Comment by cmungall** [comment]
But do they really grow inside the symbiont, rather than in the vicinity of?

**Comment by ValWood** [comment]
First, I don't think we should really try to capture the parasitism, mutualism, commensalism aspect, because it can change for a particular species in a generation.

But I'm trying to figure what these mean as processes. 

Looking at the annotations for "growth of symbiont in host"
Lots of C. albicans genes annotated  by IMP 

For example this paper
http://europepmc.org/abstract/MED/22073120
Among the 77 TF mutant strains tested, eight showed a decreased colonization including mutants for orf19.3405, orf19.255, orf19.5133, RGT1, UGA3, orf19.6182, SEF1 and orf19.2646, and four an increased colonization including mutants for orf19.4166, ZFU2, orf19.1685 and UPC2 as compared to the isogenic wild type strain. 

These are not GO processes, they are pathogen host interaction phenotypes (if we knock out this gene is the symbiont still  viable/can it infect the host). They are essentially assays to see if the gene affects viability or pathogenicity. They are not informative about the biological process.

The term  *refer* to the process 'growth'  (i.e increase in size)  but  I would bet that these have only been used for viability/proliferation of a "population".

I really think we need to remove these terms to be biologically sound.

**Comment by ValWood** [comment]
Luckily they are hardly used. ...

**Comment by ValWood** [comment]
To explain  this better.

We no longer have "organismal proliferation" in GO. We only have proliferation terms to apply to specific expansion of specific cell types in developemnt.

Proliferation is really "growth+division" (i.e viability) of a population. It encompasses all essential processes and it isn't just about an increase in size.

here we are talking about "proliferation + life cycle stage" (so an additional differentia, on top of "proliferation of a population", which is already disallowed).

We need to be careful that we  aren't using GO terms to capture viability+ lifecycle stage, because this is impossible to model  logically at the process level (every essential process would contribute).

**Comment by mgiglio99** [comment]
Above, Chris asked:
> But do they really grow inside the symbiont, rather than in the vicinity of?

My comment was referring not just to the 'growth in' terms but also to the others Pascale suggested obsoleting. 

Pascale suggested in her first post:
> These terms seem sufficient to represent what needs to be captured:
> 
> * growth of symbiont in host
> * growth of symbiont in host cell
> * growth of symbiont on or near host
> * growth of symbiont in host organelle

What I was saying was that we need more than just these - we need ones to capture the interactions where there is no host/hostee relationship. That's were the "other organism" terms came in.

But what I hear Val arguing is that even the above growth terms that Pascale noted are not allowed since they involve proliferation and not a cell getting larger. I didn't realize that cell proliferation was no longer allowed to be used outside a multicellular organism. (I've missed so much!)  :)
I would argue that the process of being able to grow in a host is not necessarily about viability, nor is it a life stage - at least not for many bacteria. Many bacteria are perfectly happy to be living in the environment (like soil) or in a host - but there will be some genes that will confer on them the ability to live in a host, that are not needed to live in the environment and vice versa.

I'm wondering if we need a "host colonization" process term - would something like that be allowed?

Michelle

**Comment by ValWood** [comment]
Yes, difficult to keep up with all the changes! and the changes to the changes, sometimes back to the original!
This book chapter by Pascale and Sylvain useful for some of the changes in annotation practice
https://link.springer.com/protocol/10.1007/978-1-4939-3743-1_4 it some  of  types of information we often used to curate, but no longer do, experimental readouts and the like.

To me  "host colonization" sounds much more like the process that  we need to capture.
Another change is to make the term labels more human readable, and to use lables that will be recognisable by curators and the community.

I'm looking under
GO:0044403    symbiont process
and many of the children seem that they might be better grouped under a "host colonization" term for example
GO:0044002    acquisition of nutrients from host
GO:0044405    recognition of host
etc...

**Comment by ValWood** [comment]
"host colonization" or similar might work as a grouping term for cases where pathogen host terms are strangely grouped in a biologically unintuitive way.

I opened a ticket for this here.
https://github.com/geneontology/go-ontology/issues/17051

**Comment by pgaudet** [comment]
**PROPOSAL** 

*  growth of symbiont in host 
-> change label to 'host colonization' 
OR to 'cell proliferation in host'? 

*  growth of symbiont in host cell
-> change label to 'host cell colonization' 
OR to 'cell proliferation in host cell' ?

*  GO:0044112 growth in other organism involved in symbiotic interaction =  0 annotations: merge somewhere ? 
*  GO:0044116   growth of symbiont involved in interaction with host  =  0 annotations: merge somewhere  : merge into 'host colonization' 
*  GO:0044125  growth of symbiont in host intercellular space  =  0 annotations: merge somewhere  : merge into 'host colonization' 
*  GO:0044121 growth of symbiont in host organelle = some annotations: change label to growth of symbiont in host vacuole (check annotation)
*     GO:0044123   growth of symbiont in host vascular tissue  = obsolete and add comment to annotate with AE 'occurs_in' [anatomical structure/cell type]



GO:0044144    modulation of growth of symbiont involved in interaction with host = 0 annotations; obsolete ? 



Obsolete ? 
 * GO:0044126 regulation of growth of symbiont in host
*  GO:0044130 negative regulation of growth of symbiont in host
*  GO:0044128 positive regulation of growth of symbiont in host 
Mostly host proteins annotated to this term

---
Equivalent Development terms: 
*    GO:0044124   development of symbiont in host intercellular space = 0 annotations 
*  GO:0044120 development of symbiont in host organelle  = 0 annotations 


Obsolete ? 
 *GO:0044139 modulation of growth of symbiont on or near host = 0 annotations
* GO:0044140 negative regulation of growth of symbiont on or near host surface = 3 annotations
* GO:0044142 positive regulation of growth of symbiont on or near host surface  = 0
Only host proteins annotated to this term

**Comment by ValWood** [comment]
I don't think we should have these term in GO:

cell proliferation in host (part of colonization): Any symbiont process that contributes to the ability of the symbiont to population expansion inside a host.
cell proliferation in host cell (part** of colonization): Any symbiont process that contributes to the ability of the symbiont to population expansion inside a cell of a host.

they would  generate exactly the same issues as the cell proliferation  term:
https://github.com/geneontology/go-ontology/issues/11478
The rationale for this was that If you try to define "proliferation" at the single celled level you always end up describing a "population phenotype" rather than a specific cellular process. We made this restriction to prevent this term being used to capture inviability phenotypes, of for capturing things like slow growth which are really only phenotypes in single celled organisms, and probably not suitable for GO annotation (i.e we knocked out this gene and we don't know what it does but the cells no longer proliferate/ proliferate more slowly)


It will be used to annotate viability/inviability phenotypes (i.e this pathogen can grow and divide in the host). This is a phenotype, and if we include it , it will be a "stub".

For example, any essential gene, or any gene required for  host colonization  would be annotated to "growth of symbiont in host"

In these instances, it is better to make no annotation if you cannot say something more specific about the process. 

If people want to annotate  population viability/inviability they should use a phenotype ontology instead.
